apps/nextjs-website/src/components/molecules/InfoCard/InfoCard.tsx

```diff
@@ -0,0 +1,34 @@
+'use client';
+import InfoCardEditButton from '@/components/atoms/InfoCardEditButton/InfoCardEditButton';
+import {
+  InfoCardItem,
+  InfoCardItemProps,
+} from '@/components/atoms/InfoCardItem/InfoCardItem';
+import { isProduction } from '@/config';
+import { Box, Card, Divider, Typography } from '@mui/material';
+
+export type InfoCardProps = {
+  cardTitle: string;
+  items: InfoCardItemProps[];
+};
+
+export const InfoCard = ({ cardTitle, items }: InfoCardProps) => {
+  const editButton = !isProduction ? <InfoCardEditButton /> : null;
+  return (
+    <Card raised sx={{ padding: 4, maxWidth: '700px' }}>
+      <Box sx={{ display: 'flex', justifyContent: 'space-between' }}>
+        <Typography variant='h6' mt={1}>
```

Comment:
The fontWeight should be `700`
![image](https://github.com/pagopa/developer-portal/assets/39835990/5ac7d569-26ba-4543-88b6-939eb963e1b2)